### PR TITLE
Messages

### PR DESCRIPTION
**Pull request by zishanmirza**
This patch removes duplicate messages.

Before submitting a PR to the `dev` branch:
- [X] Tests passing
- [X] Black formatting
- [X] Rebase/merge the `dev` branch
- [ ] Note in the CHANGELOG

### CODE REVIEW

File: dds_cli/auth.py [hunk 1]
```diff
@@ -37,9 +37,8 @@ def __init__(
     def check(self):
         token_file = user.TokenFile(token_path=self.token_path)
         if token_file.file_exists():
-            token_file.check_token_file_permissions()
```
Comment (alneberg) [inline dds_cli/auth.py:40]: Why did you remove the permissions check?

Comment (alneberg) [reply dds_cli/auth.py:40]: Ah, because it's already ran in other places!

**Comment by alneberg** [comment]
There's a call that triggers reading of the token in `__main__`:
```python
def dds_main(click_ctx, verbose, log_file, no_prompt, token_path):
    """SciLifeLab Data Delivery System (DDS) command line interface.

    Access token is saved in a .dds_cli_token file in the home directory.
    """
    # Get token metadata
    username = dds_cli.user.User.get_user_name_if_logged_in(token_path=token_path)
 ```
 As it is now it will print `Token is ok` if I do `dds ls` as well. I guess that's ok, but not sure if you intended to add that?

File: dds_cli/user.py [hunk 2]
```diff
@@ -280,56 +280,60 @@ def check_token_file_permissions(self):
...
-        elif time_to_expire < dds_cli.TOKEN_EXPIRATION_WARNING_THRESHOLD:
-            LOG.warning(
-                f"Saved token will expire in {readable_timedelta(time_to_expire)}, "
-                f"please consider renewing the session using the 'dds auth login' command."
-            )
 
         return False
 
-    def token_report(self, token):
+    def token_report(self, current_utc_time, expiration_time, log):
```
Comment (alneberg) [inline dds_cli/user.py:305]: You've removed the `token` argument here, so running e.g. `dds auth info` fails with `TypeError: TokenFile.token_report() got an unexpected keyword argument 'token'`

Comment (zishanmirza) [reply dds_cli/user.py]: I have updated the `token_report` function.

**Review by inaod568 — CHANGES_REQUESTED** [review]
> There's a call that triggers reading of the token in `__main__`:
> 
> ```python
> def dds_main(click_ctx, verbose, log_file, no_prompt, token_path):
>     """SciLifeLab Data Delivery System (DDS) command line interface.
> 
>     Access token is saved in a .dds_cli_token file in the home directory.
>     """
>     # Get token metadata
>     username = dds_cli.user.User.get_user_name_if_logged_in(token_path=token_path)
> ```
> 
> As it is now it will print `Token is ok` if I do `dds ls` as well. I guess that's ok, but not sure if you intended to add that?

@alneberg 
@zishanmirza 
I don't think we should print `Token is ok` except when `dds auth info` is run. At the moment with these changes it's not only for `dds ls` but for all commands.

The expiration time has disappeared now as well, before it was 
```
dds auth info
     ︵ 
 ︵ (  )   ︵ 
(  ) ) (  (  )   SciLifeLab Data Delivery System 
 ︶  (  ) ) (    https://delivery.scilifelab.se/ 
      ︶ (  )    Version 0.0.10 
          ︶
Current user: unituser_1
INFO     ✅  Token is OK! ✅                                                                                                                            
INFO     ✅  Token will expire in 1 day 23 hours 51 minutes! ✅    
```
and now it's
```
dds auth info
     ︵ 
 ︵ (  )   ︵ 
(  ) ) (  (  )   SciLifeLab Data Delivery System 
 ︶  (  ) ) (    https://delivery.scilifelab.se/ 
      ︶ (  )    Version 0.0.10 
          ︶
Current user: unituser_1
INFO     ✅  Token is OK! ✅    
```

**Comment by zishanmirza** [comment]
I haved fixed the issues.

**Comment by zishanmirza** [comment]
I have fixed the issues.

File: dds_cli/auth.py [hunk 3]
```diff
@@ -37,9 +37,10 @@ def __init__(
...
-            token = token_file.read_token()
-            token_file.token_report(token=token)
+            token = token_file.read_token(log=False)
+            token_file.token_report(
+                token=token, current_utc_time=None, expiration_time=None, log=True
+            )
```
Comment (inaod568) [inline dds_cli/auth.py:43]: `current_utc_time` and `expiration_time` already have the default `None` in the `token_report` function you these are not needed here

**Review by inaod568 — CHANGES_REQUESTED** [review]
At the moment the warning message isn't printed out. For example: 
```
root@601069e86a55:/code# dds ls
     ︵ 
 ︵ (  )   ︵ 
(  ) ) (  (  )   SciLifeLab Data Delivery System 
 ︶  (  ) ) (    http://dds_backend:5000/ 
      ︶ (  )    Version 0.0.10 
          ︶
Current user: unituser_1
INFO     Token expiration threshold: 1 day, 23:58:00              
INFO     Token expires in: 1 day, 23:51:50.515125                 
INFO     Warning threshold met: True                              
                         Your Project(s)                          
┏━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━┓
┃ Project   ┃           ┃         ┃           ┃ Last      ┃      ┃
┃ ID        ┃ Title     ┃ PI      ┃ Status    ┃ updated   ┃ Size ┃
┡━━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━┩
│ project_1 │ First     │ PI Name │ In        │ Mon, 07   │  0 B │
│           │ Project   │         │ Progress  │ Mar 2022  │      │
│           │           │         │           │ 13:31:56  │      │
│           │           │         │           │ UTC       │      │
│ project_2 │ Second    │ PI Name │ In        │ Mon, 07   │  0 B │
│           │ Project   │         │ Progress  │ Mar 2022  │      │
│           │           │         │           │ 13:31:56  │      │
│           │           │         │           │ UTC       │      │
└───────────┴───────────┴─────────┴───────────┴───────────┴──────┘
INFO     Would you like to view files in a specific project? Leave
         blank to exit.                                           
? Project ID:
```
This should also show the warning message about that the token expires. (Note that I have changed the threshold for this example)

The only time the warning is printed out is when the `dds auth info` is run.

**Comment by inaod568** [comment]
I'm not entirely sure of what duplication is being solved in this PR, I haven't been able to see the difference when I've tested. But the task in https://github.com/scilifelabdatacentre/dds_cli/issues/291 has been solved in https://github.com/ScilifelabDataCentre/dds_cli/pull/358, so I'm closing this now. If this is in some way incorrect or there's a duplicate message I haven't seen, we can open it again.

**Comment by zishanmirza** [comment]
Duplicated file permission checks moved to #372.
